### ISSUE TITLE
Drupal 8 : TypeError: $ is undefined

### ISSUE THREAD

**Issue by jfcartier** [almasaeed2010/AdminLTE#1487]
Hi,

I just ran into an error trying to implement AdminLTE inside a Drupal 8 Theme. It says : 

TypeError: $ is undefined (Firefox)
or
Uncaught TypeError: Cannot set property 'AdminLTE' of undefined (Chrome)

I suspect that because Drupal 8 needs a [specific way](https://www.drupal.org/docs/8/api/javascript-api/javascript-api-overview) to write javascript code, it doesnt like how it's done in app.js. 

So, here's my proposition to get it working 

```javascript
if (typeof jQuery === "undefined") {
  throw new Error("AdminLTE requires jQuery");
}

// ADD THIS LINE
var $ = jQuery.noConflict();
```

**Comment by REJack** [comment]
This errors look like that you load your js files in the wrong order.

**Comment by almasaeed2010** [comment]
@jfcartier Drupal issues have been fixed in v2.4 (tested in Drupal 7 which used to issue the same error). Please try to upgrade to v2.4 by following this guide https://adminlte.io/docs/2.4/upgrade-guide

If the issue persists, please open a new issue.

We greatly appreciate your contribution! Thanks.
